load old deprecated wrappers, one more release, some plugins still have
addInitEvent() calls

https://www.dokuwiki.org/devel:jqueryfaq
--- dokuwiki/lib/exe/js.php~	2013-05-28 16:12:49.000000000 +0300
+++ dokuwiki/lib/exe/js.php	2013-05-28 21:25:23.721168974 +0300
@@ -62,7 +62,7 @@
                 DOKU_INC.'lib/scripts/locktimer.js',
                 DOKU_INC.'lib/scripts/linkwiz.js',
                 DOKU_INC.'lib/scripts/media.js',
-# deprecated                DOKU_INC.'lib/scripts/compatibility.js',
+                DOKU_INC.'lib/scripts/compatibility.js',
 # disabled for FS#1958                DOKU_INC.'lib/scripts/hotkeys.js',
                 DOKU_INC.'lib/scripts/behaviour.js',
                 DOKU_INC.'lib/scripts/page.js',
